packages/config/config/devices/0x031e/lzw31-sn.json

```diff
@@ -89,83 +88,81 @@
 		},
 		{
 			"#": "5",
-			"label": "Minimum Dim Level",
-			"unit": "%",
-			"valueSize": 1,
-			"minValue": 1,
-			"maxValue": 45,
-			"defaultValue": 1
+			"$import": "~/templates/master_template.json#minimum_dim_level_1-99",
+			"maxValue": 45
```

Comment:
I'm not 100% happy with this approach (importing a template with a named min/max and then changing the limits). Probably just leave it as is.